common/connlimit/config_test.go

```diff
@@ -18,12 +18,13 @@
 package connlimit
 
 import (
-	. "github.com/smartystreets/goconvey/convey"
 	"testing"
 	"time"
+
+	. "github.com/smartystreets/goconvey/convey"
```

Comment:
Any specific reason to use dot import here ?

Reply:
Your test code was written like this before. I took a look, it should be because you want to use Convey directly, so you used the dot import method